### PR TITLE
changes to swb import

### PR DESCRIPTION
**Pull request by robmsmt**
https://github.com/mozilla/DeepSpeech/issues/734

### CODE REVIEW

File: bin/import_swb.py [hunk 1]
```diff
@@ -13,32 +17,88 @@
 import unicodedata
 import wave
 import codecs
+import tarfile
+#from util.downloader import maybe_download, SIMPLE_BAR
```
Comment (kdavis-mozilla) [inline bin/import_swb.py]: Left overs? This should be removed.

File: bin/import_swb.py [hunk 2]
```diff
@@ -13,32 +17,88 @@
...
 
     # Write sets to disk as CSV files
-    train_files.to_csv(os.path.join(data_dir, "swb-train.csv"), index=False)
-    dev_files.to_csv(os.path.join(data_dir, "swb-dev.csv"), index=False)
-    test_files.to_csv(os.path.join(data_dir, "swb-test.csv"), index=False)
+    train_files.to_csv(os.path.join(target_dir, "swb-train.csv"), index=False)
+    dev_files.to_csv(os.path.join(target_dir, "swb-dev.csv"), index=False)
+    test_files.to_csv(os.path.join(target_dir, "swb-test.csv"), index=False)
+
+def _extract(target_dir, archive_path):
```
Comment (kdavis-mozilla) [inline bin/import_swb.py]: What if the file was already extracted? For example, the import failed in the middle, but completed the extraction.

This should not extract a second time.

File: bin/import_swb.py [hunk 3]
```diff
@@ -13,32 +17,88 @@
...
+    new_data_dir = os.path.join(data_dir, ARCHIVE_DIR_NAME)
+    target_dir = os.path.abspath(new_data_dir)
+    archive_path = os.path.abspath(os.path.join(data_dir, LDC_DATASET))
+
+    # Check swb1_LDC97S62.tgz then extract
+    assert(os.path.isfile(archive_path))
+    _extract(target_dir, archive_path)
+    
+    # Transcripts
+    transcripts_path = maybe_download(ARCHIVE_URL, target_dir, ARCHIVE_NAME)
```
Comment (kdavis-mozilla) [inline bin/import_swb.py]: Nice! Downloading the ISIP corrected transcripts.

My one question is: How do the transcripts you refer to a http://www.openslr.org/resources/5/ differ from the official ISIP ones [here](https://www.isip.piconepress.com/projects/switchboard/)?

Could you make sure this downloads the newest transcripts and also add a comment where `ARCHIVE_NAME` is defined indicating which version of the ISIP transcript is being downloaded.

File: bin/import_swb.py (hunk 1 above)
Comment (robmsmt) [reply bin/import_swb.py]: Thanks, I have removed this, added pep8 spacing between functions and added a better run comment.

File: bin/import_swb.py (hunk 3 above)
Comment (robmsmt) [reply bin/import_swb.py]: They are identical it seems:

```
rob@rob-tpt470s:~/Downloads$ diff switchboard_word_alignments.tar.gz switchboard_word_alignments_openslr.tar.gz 
rob@rob-tpt470s:~/Downloads$ crc32 switchboard_word_alignments.tar.gz switchboard_word_alignments_openslr.tar.gz 
c0765d32	switchboard_word_alignments.tar.gz
c0765d32	switchboard_word_alignments_openslr.tar.gz
```

Comment (kdavis-mozilla) [reply bin/import_swb.py]: As I need to built on this result now, I've opened a new issue #1747 for this last point on the ISIP transcript version, and will merge this code and fix #1747 in another PR.

**Pull request merged**